awesome/views/movie/movies.hbs

```diff
@@ -0,0 +1,14 @@
+<h2>Ironhack Cinema</h2>
+<h3>Click on the movie to check the showtimes!</h3>
+
+<ul>
+    {{#each movies}}
+        <li>
+            <div>
+              <img src="{{this.image}}">
+              <h3>{{this.title}}</h3>
+              <a href="movie/:id">See more</a>
```

Comment:
`<a href="movie/{{this._id}}">See more</a>`
Hay que poner el ID de la peli con this._id